docs/use_cases/model_applications/marine_and_cryosphere/GridStat_fcstRTOFS_obsSMAP_climWOA_sss.py

```diff
@@ -101,6 +101,8 @@
 # If there is a setting in the MET configuration file that is currently not supported by METplus you'd like to control, please refer to:
 # :ref:`Overriding Unsupported MET config file settings<met-config-overrides>`
 #
+# .. dropdown:: GridStatConfig_wrapped
+#
 # .. note:: See the :ref:`GridStat MET Configuration<grid-stat-met-conf>` section of the User's Guide for more information on the environment variables used in the file below:
 #
 # .. highlight:: bash
```

Comment:
```suggestion
```
Please remove this line as the template does not include it. Please ensure that the literal include beneath this line is indented for the dropdown.